### PR TITLE
Add requirements.txt support

### PR DESCRIPTION
**Pull request by CrispyBacon1999**
Add documentation to match the syntax of https://github.com/robotpy/robotpy-installer/pull/33.

### CODE REVIEW

File: install/packages.rst [hunk 1]
```diff
@@ -184,6 +184,22 @@ download-opkg
 Downloads an ipk file from the RobotPy and NI's online opkg repositories, along
 with the dependencies for the package.
 
+You can also use a `requirements.txt` file to specify which packages should be downloaded
+::
+
+  robotpy-installer download-opkg -r requirements.txt
+
+This file matches the syntax of a standard pip `requirements.txt`.
```
Comment (auscompgeek) [inline install/packages.rst]: Ehh… pip's requirements.txt is a fair bit more complex than what we have here (notably no version constraints or nesting of requirements files). Unless if you plan to fix that in the near future, we can just say it's a list of packages.

File: install/packages.rst [hunk 2]
```diff
@@ -194,6 +210,13 @@ install-opkg
 Copies ipk files over to the roboRIO, and installs them and their dependencies.
 If the package already has been installed, it will do nothing.
 
+You can also use a `requirements.txt` file to specify which packages should be downloaded
+::
+
+  robotpy-installer download-opkg -r requirements.txt
+
+
+
```
Comment (auscompgeek) [inline install/packages.rst]: This whitespace seems excessive.

File: install/packages.rst [hunk 3]
```diff
@@ -184,6 +184,22 @@ download-opkg
 Downloads an ipk file from the RobotPy and NI's online opkg repositories, along
 with the dependencies for the package.
 
+You can also use a `requirements.txt` file to specify which packages should be downloaded
```
Comment (auscompgeek) [inline install/packages.rst]: This sentence is lacking punctuation… you could join the next line to make it end with a colon.

File: install/packages.rst [hunk 4]
```diff
@@ -194,6 +210,13 @@ install-opkg
 Copies ipk files over to the roboRIO, and installs them and their dependencies.
 If the package already has been installed, it will do nothing.
 
+You can also use a `requirements.txt` file to specify which packages should be downloaded
```
Comment (auscompgeek) [inline install/packages.rst]: Lacking punctuation.

File: install/packages.rst (hunk 1 above)
Comment (CrispyBacon1999) [reply install/packages.rst]: Changed the wording of it a bit, should be a lot more accurate now.

File: install/packages.rst [hunk 5]
```diff
@@ -184,6 +184,21 @@ download-opkg
 Downloads an ipk file from the RobotPy and NI's online opkg repositories, along
 with the dependencies for the package.
 
+You can also use a `requirements.txt` file to specify which packages should be downloaded:
+::
```
Comment (auscompgeek) [inline install/packages.rst]: Having a colon at the end of the previous line and this double-colon on its own line is a bit redundant.

Comment (CrispyBacon1999) [reply install/packages.rst]: Didn't realize that the double colon would also add a colon to the previous line. RST is a bit weird sometimes.

**Pull request merged**